icu4c/source/data/pkgdata.osx.inc

```diff
@@ -0,0 +1,17 @@
+GENCCODE_ASSEMBLY_TYPE=-a gcc-darwin
```

Comment:
It would be nice to have a TODO here saying some of these flags may not be needed at all and needs some cleaning.